compute/compute-node.Dockerfile

```
@@ -1527,6 +1527,51 @@ COPY --from=pgauditlogtofile-src /ext-src/ /ext-src/
 WORKDIR /ext-src/pgauditlogtofile-src
 RUN make install USE_PGXS=1 -j $(getconf _NPROCESSORS_ONLN)
 
+#########################################################################################
+#
+# Layer "pg_rest-build"
+# compile pg_rest extension
+#
+#########################################################################################
+FROM build-deps AS pg_rest-src
+ARG PG_VERSION
+ARG PG_REST_VERSION=3.0.1
+
+# Only supported for PostgreSQL v17
+RUN if [ "${PG_VERSION:?}" != "v17" ]; then \
+        echo "pg_rest extension is only supported for PostgreSQL v17" && exit 0; \
+    fi
+
+WORKDIR /ext-src
+RUN mkdir -p pg_rest-src && cd pg_rest-src && \
+    wget https://github.com/ruslantalpa/foxfirebase/raw/main/pg_rest_pg17-${PG_REST_VERSION}_neon-debian-bookworm_aarch64.deb -O pg_rest_pg17-${PG_REST_VERSION}_aarch64.deb && \
```

```suggestion
    wget https://github.com/ruslantalpa/foxfirebase/blob/120d9b71fc39480890c2b46d21e0ff606e374406/pg_rest_pg17-${PG_REST_VERSION}_aarch64.deb
    wget https://github.com/ruslantalpa/foxfirebase/raw/main/pg_rest_pg17-${PG_REST_VERSION}_neon-debian-bookworm_aarch64.deb -O pg_rest_pg17-${PG_REST_VERSION}_aarch64.deb && \
```

let's pin to specific commit. I'd also consider additional `sha256sum` check, per analogy to https://github.com/neondatabase/neon/blob/c66444ea1538349d13ab5e87bca880394434004b/compute/compute-node.Dockerfile#L1309

---

I'd also consider forking this repo and `wget`ing form our fork